src/components/title.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import * as fonts from '../assets/fonts'
+
+type TitleSize = "small" | "medium" | "large" | "xlarge" | "xxlarge"
+
+interface TitleProps extends React.HTMLProps<JSX.Element> {
+  titleSize?: TitleSize
+}
+
+const titleSizes = {
+  small: '25px',
+  medium: '30px',
+  large: '37px',
+  xlarge: '50px',
+  xxlarge: '72px'
+}
+
+const Title: React.SFC<TitleProps> = props => (
+  <div className={props.className}>
```

Comment:
It does have support for `h1-h6` tags or any html tags for that matter. We thought about using them for semantic reasons but decided to follow how titles are implemented in elan which uses `p` tags for all sizes. `div` would work just the same